docs/source/installing-brave.rst

```diff
@@ -166,6 +166,22 @@ Fedora 28+
 
     sudo dnf install brave-browser-dev
 
+
+Centos/RHel
+----------
+::
+
+    sudo rpm --import  https://brave-browser-rpm-dev.s3.brave.com/brave-core-nightly.asc
+
+    sudo cat << EOF >  /etc/yum.repos.d/Brave.repo
+    [brave]
+    name=Brave Browser repo
```

Comment:
Brave Browser Dev repo

Reply:
This is code I rebased in . It's removed now